content/java/concepts/math-methods/terms/tan/tan.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.tan()'
+Description: 'Returns the tangent of an angle given in radians.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Function'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.tan()`** method returns tangent of an angle given in radians. The function accepts all real numbers *except* the values where `cos(x)` is equal to 0  
```

Comment:
"...returns *the* tangent of an angle given in radians."
"...is equal to zero."